go.mod

```diff
@@ -67,3 +67,17 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
```

Comment:
@msau42 Should I delete these replace statements here?

Reply:
If the script is adding it then let's keep it. We should investigate how to modify the script to remove it if it's unnecessary.